web/deploy/terraform/modules/shared_resources/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "aws_region" {
+  description = "AWS region"
+  default     = "us-east-1"
+}
+
+
```

Comment:
Another stylistic suggestion: have only one empty line instead of two between variable definition

Reply:
Done